libs/SalesforceHybrid/build.gradle

```diff
@@ -15,7 +15,7 @@ apply plugin: 'com.jfrog.bintray'
 
 dependencies {
   api project(':libs:SmartSync')
-  api 'org.apache.cordova:framework:6.4.0:release@aar'
```

Comment:
What about the release@aar

Reply:
Looks like they uploaded it the right way this time, so we don't need it :P

Reply:
It'll fail with it.